### PR TITLE
es6-template-literals | Kianna Brandt

### PR DESCRIPTION
**Pull request by KiannaBrandt**
Screen Shot 2021-04-17 at 12 24 45 PM

### CODE REVIEW

File: es6-template-literals/index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Javascript ES6 - Template Literals</title>
+</head>
+<body>
+  <script>
+    const number = 9200;
+    const street = ' Irvine Center Drive';
+    const address = number + street;
```
Comment (TimelikeClosure) [inline es6-template-literals/index.html]: You missed a concatenation.

---

Screenshot_3

File: es6-template-literals/index.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+    console.log(`LearningFuze: ${address}`);
```
Comment (TimelikeClosure) [inline es6-template-literals/index.html]: The original statement didn't have a concatenation:

---

Screenshot_4

File: es6-template-literals/index.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+    const firstName = 'Uzair';
+    const lastName = 'Ashraf';
+    const age = 25;
+
+    const bio = `My name is ${firstName} ${lastName} and I am ${age} years old.`;
+
+    console.log(`Bio: ${bio}`);
```
Comment (TimelikeClosure) [inline es6-template-literals/index.html]: The original statement didn't have a concatenation:

---

Screenshot_5

File: es6-template-literals/index.html [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+    const fruit = 'apple';
+    const item = 'pen';
+
+    const applePen = fruit + item;
```
Comment (TimelikeClosure) [inline es6-template-literals/index.html]: You missed a concatentation.

File: es6-template-literals/index.html [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+    console.log(`I have a ${item}`);
+    console.log(`I have an ${fruit}`);
```
Comment (TimelikeClosure) [inline es6-template-literals/index.html]: These statements didn't originally have a concatentation:

---

Screenshot_6

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

**Comment by KiannaBrandt** [comment]
Screen Shot 2021-04-17 at 12 58 45 PM

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**